infra/base-images/base-runner/Dockerfile

```diff
@@ -37,6 +37,7 @@ RUN apt-get update && apt-get install -y \
     python3 \
     python3-pip \
     wget \
+    curl \
```

Comment:
Should be there already now.

Reply:
Same: I see it in base-builder but not base-runner...

Reply:
Should it be moved to base-image or some other generic image ?